### PR TITLE
Add support for POST on policies

### PR DESCRIPTION
**Pull request by svt-hchaudhari**
Signed-off-by: Hemant Chaudhari <hchaudhari@hpe.com>

### CODE REVIEW

File: examples/policies.py [hunk 1]
```diff
@@ -70,3 +70,14 @@
 vms = policy.get_vms()
 print(policy.data)
 print(vms)
+
+print("\n\n create policy")
+policy_name = "test_policy"
+policies.create(policy_name)
```
Comment (svt-kmcdaniel) [inline examples/policies.py]: should we use the return from policies.create to delete the policy?

That will ensure that we are deleting the policy that was created.

File: simplivity/resources/policies.py [hunk 2]
```diff
@@ -73,6 +73,20 @@ def get_by_data(self, data):
...
+        Args:
+            flags: Dictionary of filters, example: {'cluster_group_id': 'cluster_group_id'}
+            name : name of policy
+            timeout : timeout for the operation to complete
+
+        Returns:
+            Policy object.
+        """
+        data = {"name": name}
+        return self._client.do_post(URL, data, timeout, None, flags)
```
Comment (svt-kmcdaniel) [inline simplivity/resources/policies.py]: Are you sure that do_post returns a policy object?

I thought do_post returns a list of affected objects.

File: tests/unit/resources/test_policies.py [hunk 3]
```diff
@@ -141,6 +141,16 @@ def test_get_vms_not_found(self, mock_get):
 
         mock_get.assert_called_once_with('/policies/ABCDE/virtual_machines')
 
+    @mock.patch.object(Connection, "post")
+    def test_create_policy(self, mock_post):
+        mock_post.return_value = None, [{'name': 'test'}]
+        policy_name = 'test'
+        self.policies.create(policy_name)
```
Comment (svt-kmcdaniel) [inline tests/unit/resources/test_policies.py]: need to ensure that policies.create() returns a Policy object within the unit tests

**Review by svt-kmcdaniel — CHANGES_REQUESTED** [review]

File: examples/policies.py (hunk 1 above)
Comment (svt-hchaudhari) [reply examples/policies.py]: Yes made the respective changes to get the list of policies and then retrieving the required policy object to delete the same.

File: simplivity/resources/policies.py (hunk 2 above)
Comment (svt-hchaudhari) [reply simplivity/resources/policies.py]: You are right it does returns the affected objects and from there we shall retrieve the policy object and send it back as part of policies.

File: tests/unit/resources/test_policies.py (hunk 3 above)
Comment (svt-hchaudhari) [reply tests/unit/resources/test_policies.py]: Agreed, made the respective changes.

**Pull request closed without merging**